themes/theme-gmd/pages/ProductGallery/components/Content/components/ImagesSlider/index.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Swiper } from '@shopgate/pwa-common/components';
```

Comment:
Could you please change this import to you `@shopgate/engage/components`, please?